NitroxClient/MonoBehaviours/IntroCinematicUpdater.cs

```diff
@@ -0,0 +1,164 @@
+
+    private IEnumerator Start()
+    {
+        remoteRenders.ForEach(smr => smr.enabled = false);
+        seatBarRendererRight.ForEach(smr => smr.updateWhenOffscreen = true);
+
+        yield return new WaitForSecondsRealtime(11f); //Local player in seat
+        remoteRenders.ForEach(smr => smr.enabled = true);
+
+        yield return new WaitForSecondsRealtime(33.45f - 11f); //Remote Player deadly hit => disable original head
+        remotePlayerHeadRenderer.enabled = false;
+
+        yield return new WaitForSecondsRealtime(33.58f - 33.45f); //Remote Player visible => enable custom head
+        remotePlayerCustomHead.SetActive(true);
+
+        yield return new WaitForSecondsRealtime(40f - 33.58f); //Seatbelt appears => enable position sync & switch to original head
+        remotePlayerCustomHead.SetActive(false);
+        remotePlayerHeadRenderer.enabled = true;
+        Destroy(remotePlayerCustomHead);
+
+        yield return new WaitForSecondsRealtime(50.5f - 40f); //Slowly move remote player backwards
+        shouldDisplaceRemotePlayer = true;
+
+        yield return new WaitForSecondsRealtime(50.72f - 50.5f); //Disable and move remote player
+        shouldDisplaceRemotePlayer = false;
+        remotePlayerDisplacement = Vector3.zero;
+        remoteRenders.ForEach(smr => smr.enabled = false);
+
+        yield return new WaitForSecondsRealtime(53f - 50.72f); //Slowly move local player to modified cin-end
+        shouldDisplaceLocalPlayer = true;
+    }
+
+    private void LateUpdate()
+    {
+        Vector3 modelRootPos = modelRoot.position;
+        Quaternion modelRootRot = modelRoot.rotation;
+
+        // Coping the global rotation, inverting the y and adding 180° to it
+        Quaternion InverseRotateAroundEscapePod(Quaternion from)
+        {
+            Quaternion offset = Quaternion.Inverse(modelRootRot) * from;
+            return modelRootRot * rotateOtherSide * new Quaternion(-offset.x, offset.y, offset.z, -offset.w);
+        }
+
+        // Mirror animate the remote like the local player
+        Vector3 posDiff = playerTransform.position - modelRootPos;
+        remotePlayerDestinationPos = modelRootPos + new Vector3(posDiff.x, posDiff.y, -posDiff.z) + remotePlayerDisplacement;
+        remotePlayerDestinationRot = InverseRotateAroundEscapePod(playerTransform.rotation);
+        transform.SetPositionAndRotation(remotePlayerDestinationPos, remotePlayerDestinationRot);
+
+        // Mirror animate the seat parts and stop it before the animation ends so the SN animator resets our manipulations
+        if (!shouldDisplaceLocalPlayer)
+        {
+            for (int i = 0; i < seatPartsLeft.Length; i++)
+            {
+                seatPartsRight[i].localRotation = seatPartsLeft[i].localRotation;
+            }
+
+            seatArmRestRight.localPosition = -seatArmRestLeft.localPosition;
+        }
+
+        if (shouldDisplaceRemotePlayer) remotePlayerDisplacement += new Vector3(0, 0, 0.05f);
```

Comment:
There should be brackets here. Per contribution guidelines: "Encapsulate everything in braces, even single-line statements."

Also trying to figure out what this is doing for my own learning, why are we displacing by this amount? This seems too tiny to be the distance between the seats so I'm confused what its purpose is.

Reply:
We updated our guidelines and code style internally but didn't published this change to the contribution guidelines. This should now be updated 👍